lib/project/ext/object.rb

```diff
@@ -10,4 +10,9 @@ def device
   def find(*args) # Do not alias this, strange bugs happen where classes don't have methods
     rmq(*args)
   end
+
+  def find!(*args)
+    #TODO this can be rmq.find!(*args) when released in RMQ
```

Comment:
No, `rmq.find!` is different. `rmq(a).find!(b)` <- this find is for all children/grandchildren.
This find is: rmq(a).get. I'd change that to this too, btw, no need for one extra method call.

Thanks David.